src/ws-inline-edit/ws-inline-edit.js

```diff
@@ -100,7 +91,13 @@ export class WSInlineEdit extends Component {
    */
   render() {
     return (
-      <div className="ws-inline-edit" onClick={() => this.editElement()}>
+      <div
+        className="ws-inline-edit"
+        onClick={() => this.editElement()}
+        onKeyPress={e => {
```

Comment:
I would say a keyPress on a non selectable element makes no sense or?
I never can tab to this element until a tabindex is making it focusable. Does a tabindex make it focusable even?^^

Reply:
But there is no tabindex or I'm just not seeing it? Anyways I think adding the tabindex 0 isn't right but I have to re-check it.